packages/storybook/src/nimble/tabs/tabs.stories.ts

```diff
@@ -96,6 +102,13 @@ export const tabs: StoryObj<TabsArgs> = {
         ${repeat(x => (tabSets[x.tabsType] as TabArgs[]), html<TabArgs>`
             <${tabPanelTag}>Content of tab ${x => x.id}</${tabPanelTag}>
         `, { positioning: true })}
+        ${when(x => x.tabsType === ExampleTabsType.simpleTabsWithToolbar, html<TabsArgs>`
+            <${tabsToolbarTag}>
+                <${buttonTag} appearance="ghost">Toolbar Button</${buttonTag}>
+                <${buttonTag} appearance="ghost" slot="end">Toolbar Button 2</${buttonTag}>
+                <${buttonTag} appearance="ghost" slot="end">Toolbar Button 3</${buttonTag}>
+            </${tabsToolbarTag}>
+        `)}
         </${tabsTag}>
     `),
     argTypes: {
```

Comment:
Same as above, can you update the labels in tabsType below to give pretty display version like the others:
![image](https://github.com/user-attachments/assets/bf3ca261-66eb-47d3-921f-a662f0e21f22)